install.sh

```diff
@@ -107,6 +107,10 @@ then
   export USER
 fi
 
+# Allow delegating installation to a different user (useful for installs by root)
+INSTALL_USER="${INSTALL_USER-${USER}}"
+INSTALL_HOME=$(eval echo ~"${INSTALL_USER}")
```

Comment:
This will quietly fail if the user does not exist.

Reply:
I'll add a validation for the INSTALL_USER to exist